### ISSUE TITLE
Dataset?

### ISSUE THREAD

**Issue by mrgloom** [yiranran/Unpaired-Portrait-Drawing#8]
In paper `"We collect face photos and APDrawings from the Internet and construct a training corpus of 798 face photos and 625 delicate portrait line drawings, and a test set of 154 face photos."` is this crawled web dataset available?

**Comment by yiranran** [comment]
sorry, we could not provide training data because of copyright problems...
